inform7/Internal/Inter/Architecture32Kit/Sections/Glk.i6t

```diff
@@ -882,6 +882,81 @@ Constant wintype_TextGrid = 4;
   return ret;
 ];
 
+@ |GGRecoverObjects| handles recovering the Glk objects after restarting or restoring.
+
+=
+[ GGRecoverObjects id;
```

Comment:
Moved because it's really part of the Glk implementation rather than a proper startup function. When the Glk foundations are added, then this function will be rewritten, and it will make sense for it to live in the same file as the rest of what will be added.